### PR TITLE
updated content

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/installed-collectors/collector-installation-reference/collector-installation-error-messages.md [hunk 1]
```diff
@@ -85,3 +84,24 @@ Instead, you must use an access ID/access key to register the Collector.
...
+                        Extending timeouts.
+INFO   | jvm 1    | JVM Process has not received any CPU time for x seconds.
+                        Extending timeouts.
+```
+
+These messages are warnings that, in this case, both the Wrapper and its JVM process were denied access to the CPU for a period of x seconds.
+
+To solve this issue and improve the resilience of the Wrapper and its JVM process when facing a lack of CPU, you can modify the `wrapper.conf` file by adding the following configurations at the end.
+
+```wrapper.cpu.timeout=85
```
Comment (kimsauce) [inline docs/send-data/installed-collectors/collector-installation-reference/collector-installation-error-messages.md:103]: Need to press "Return" between the first three backticks and `wrapper.cpu.timeout=85`.

**Review by kimsauce — DISMISSED** [review]
minor code block formatting fix

File: docs/send-data/installed-collectors/collector-installation-reference/collector-installation-error-messages.md (hunk 1 above)
Comment (JainM6) [reply docs/send-data/installed-collectors/collector-installation-reference/collector-installation-error-messages.md]: Thanks, Kim. That was very specific 🥇

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
looks great!

**Pull request merged**